frontend/src/components/Post.js

```diff
@@ -1,79 +1,66 @@
+        setRefreshComment(!refreshComment);
     }
 
-    render() {
-        return (
-            <div>
-                {this.state.Picture ?
-                    <div className="post">
-                        <div className="image-wrapper">
-                            <img src={`${global.config.BACKEND_URL}/picture/${this.props.id}.jpg`}
-                                alt={`${this.props.id}`} />
-                        </div>
-                        <div className="text-wrapper">
-                            {/* a Description component can be created to facilitate the creation of Post components */}
-                            <div className="post-description">
-                                <Link to={`/account/${this.state.Picture.account}`}>
-                                    {this.state.Picture.account}
-                                </Link>: {this.state.Picture.caption}
-                            </div>
-                            <div className="comments">
-                                <CommentList refreshComment={this.state.refreshComment} postId={this.props.id} />
-                            </div>
-                            <div className="postsComment">
-                                <PostComment postId={this.props.id} onCommentPosted={this.onCommentPosted.bind(this)} />
+    return (
+        <div>
+            {Picture ?
+                <div className="post">
+                    <div className="image-wrapper">
+                        <img src={`${global.config.BACKEND_URL}/picture/${props.id}.jpg`}
+                            alt={`${props.id}`} />
+                    </div>
+                    <div className="text-wrapper">
+                        {/* a Description component can be created to facilitate the creation of Post components */}
+                        <div className="post-description">
+                            <div className="account-name">
+                                <Link to={`/account/${Picture.account}`}>
+                                    {Picture.account}
+                                </Link>: {Picture.caption}
                             </div>
+
+                        </div>
+                        <div className="comments">
+                            <CommentList refreshComment={refreshComment} postId={props.id} />
+                        </div>
+                        <div className="posts-Comment">
+                            <PostComment postId={props.id} onCommentPosted={onCommentPosted} />
                         </div>
                     </div>
-                    :
-                    this.state.Picture === undefined ? null : <div className="error">The picture could not be found.</div>
-                }
-            </div>
-        );
-    }
+                </div>
+                :
+                Picture === undefined ? null : <div className="error">The picture could not be found.</div>
```

Comment:
Little trick: instead of putting `A === undefined ? null : B` I'm pretty sure you can simply do `A && B` and it will have the same behaviour, wherein if A is undefined then that part of code just won't render, else it will display 'B'

Reply:
That's pretty cool